Utilities/macrecovery/macrecovery.py

```diff
@@ -208,9 +272,10 @@ def action_download(args):
     print(info)
   print('Downloading ' + info[INFO_PRODUCT] + '...')
   dmgname = '' if args.basename == '' else args.basename + '.dmg'
-  save_image(info[INFO_IMAGE_LINK], info[INFO_IMAGE_SESS], dmgname, args.outdir)
+  dmgname = save_image(info[INFO_IMAGE_LINK], info[INFO_IMAGE_SESS], dmgname, args.outdir)
```

Comment:
Feels like dmgpath to me? Name is generally just the filename.

Reply:
Renamed *name -> *path variables for `verify_image`.